tools/biobb_pytorch/biobb_train_mdae.xml

```diff
@@ -0,0 +1,109 @@
+<tool id="biobb_pytorch_train_mdae" name="TrainMdae" version="@TOOL_VERSION@" profile="22.05">
+    <description>Train a Molecular Dynamics AutoEncoder (MDAE) PyTorch model.</description>
+    <macros>
+      <token name="@TOOL_VERSION@">4.2.1</token>
+    </macros>
+
+    <requirements>
+      <requirement type="package" version="@TOOL_VERSION@">biobb_pytorch</requirement>
+    </requirements>
+
+    <command detect_errors="exit_code"><![CDATA[
+
+      `#Required inputs using file extension`
+      ln -s '$input_train_npy_path' ./input_train_npy_path.$input_train_npy_path.ext &&
+
+      `#Optional inputs using file extension`
+      #if $input_model_pth_path:
+        ln -s '$input_model_pth_path' ./input_model_pth_path.$input_model_pth_path.ext &&
+      #end if
+      #if $config_json:
+        ln -s '$config_json' ./config_json.$config_json.ext &&
+      #end if
+
+      `#Required Outputs using file extension`
+      touch '$output_model_pth_path' &&
+      ln -s '$output_model_pth_path' ./output_model_pth_path.pth &&
+
+      `#Optional Outputs using file extension`
+      #if $output_train_data_npz_path:
+        touch '$output_train_data_npz_path' &&
+        ln -s '$output_train_data_npz_path' ./output_train_data_npz_path.npz &&
+      #end if
+      #if $output_performance_npz_path:
+        touch '$output_performance_npz_path' &&
+        ln -s '$output_performance_npz_path' ./output_performance_npz_path.npz &&
+      #end if
```

Comment:
Same as above. I never saw that outputs needs to be `touched` or linked.

Reply:
Thank you for your feedback. I removed the part you mentioned, used a literal in the command, and utilized the "from_work_dir" attribute. This has significantly simplified things.